internal/spentenergy/spentenergy.go

```diff
@@ -13,17 +14,50 @@ const (
 )
 
 func WalkingSpentCalories(steps int, weight, height float64, duration time.Duration) (float64, error) {
-	// TODO: реализовать функцию
+
+	if duration <= 0 || weight <= 0 || height <= 0 || steps <= 0 {
+		return 0, fmt.Errorf("Ошибка параметров в WSC")
```

Comment:
Через месяц уже не вспомнишь, что это за аббревиатура, ctrl+c ctrl+v
```suggestion
		return 0, fmt.Errorf("Ошибка параметров в WalkingSpentCalories")
```